mongo/options/encryptoptions.go

```diff
@@ -81,7 +82,7 @@ func (e *EncryptOptions) SetContentionFactor(contentionFactor int64) *EncryptOpt
 	return e
 }
 
-// SetRangeOptions specifies the options to use for explicit encryption with range. It is only valid to set if algorithm is "rangePreview".
+// SetRangeOptions specifies the options to use for explicit encryption with range. It is only valid to set if algorithm is "range".
```

Comment:
Optional: Add a blank line here to make the "Beta" warning more visible.
```suggestion
// SetRangeOptions specifies the options to use for explicit encryption with range. It is only valid to set if algorithm is "range".
//
```